protocols/mdns/Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "libp2p-mdns"
 edition = "2018"
-version = "0.33.0"
+version = "0.33.1"
```

```suggestion
version = "0.34.0"
```

While not a breaking change on the API level, it is on the network level. Using a patch release here would make downstream dependents auto-update. Given that this is a breaking change and might split a network, I would prefer it to be an explicit decision to update.

Could you bump the version of `libp2p-mdns` in the root `Cargo.toml` and mention the update in the root `CHANGELOG.md` file as well?